bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
     features.operators.openshift.io/token-auth-aws: "false"
     features.operators.openshift.io/token-auth-azure: "false"
     features.operators.openshift.io/token-auth-gcp: "false"
-    olm.skipRange: '>=9.0.0 <10.15.0'
+    olm.skipRange: '>=9.0.0 <10.15.1'
```

Comment:
Please add the index change as well https://github.com/openshift/windows-machine-config-operator/blob/d40284976de0c015177706f7174999ec6bbac03f/bundle/metadata/annotations.yaml#L12

Reply:
addressed